src/components/FormControlLabel/src/FormControlLabel.stories.tsx

```diff
@@ -19,7 +19,7 @@ export default {
 } as Meta;
 
 const Template: Story<FormControlLabelProps> = (args: FormControlLabelProps) => (
-  <FormControlLabel label="Checkbox" control={<Checkbox {...args} />} {...args} />
+  <FormControlLabel {...args} label="Checkbox" control={<Checkbox {...args} />} />
```

Comment:
Why was this ordering changed?

Reply:
If left as before, typescript considers label and control to be specified twice (once in ...args) and gives an error.
Swapping them like this causes label and control to 'override' the one in {...args} and no error occurs.